MvvmCross/Platforms/Android/Presenters/Attributes/MvxFragmentPresentationAttribute.cs

```diff
@@ -90,6 +94,12 @@ public MvxFragmentPresentationAttribute()
         /// </summary>
         public bool AddToBackStack { get; set; } = DefaultAddToBackStack;
 
+        public static bool DefaultAllowOnlyOneInstance = false;
+        /// <summary>
+        /// Allow only one instance at a time. False by default.
+        /// </summary>
+        public bool AllowOnlyOneInstance { get; set; } = DefaultAllowOnlyOneInstance;
```

Comment:
I think we might need to rethink this a bit. IsCacheableFragment should already indicate if it is a single instance or reused. Because this problem exists on all platforms i was thinking to make 2 new properties in a base class:

```
public bool CacheView = false;
public bool CacheViewModel = false;
```
We could then implement this over all the platforms and make a stable way to cache views and viewmodels.

Reply:
@martijn00 which class would these properties reside?